lib/get-commit.js

```diff
@@ -17,7 +17,8 @@ function getCommit (state) {
       message: result.data.commit.message,
       filename,
       patch,
-      branchUrl: branchUrl
+      branchUrl: branchUrl,
+      committer: result.data.author.login
```

Comment:
Instead of `commiter` I would suggest to set `authorLogin`. 

Author is the person who created the commit, Committer is the person you added the commit to the commit history. See https://github.com/hoodiehq/first-timers-bot/commit/c64b40f918d33263e3c6156683c4bec37d4924ab for example, in that commit, you are the author, I am the committer.  Here is the object you'd receive from the API https://api.github.com/repos/hoodiehq/first-timers-bot/git/commits/c64b40f918d33263e3c6156683c4bec37d4924ab

And just `author` would imply an object, while `authorLogin` is unambiguous